.gitignore

```diff
@@ -11,4 +11,16 @@ __pycache__
 
 #IntelliJ
 .idea
-*.iml
\ No newline at end of file
+*.iml
+
+#Setup
+MANIFEST
+build/
```

Comment:
No... MANIFEST stays.  The ones below it should go.  So `build/` and `dist/` and
`scudcloud.egg-info/`